examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "# 04 - How to do FWI with Dask\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import dask\n",
```

Comment:
This is not a hard requirement of Devito. We should catch the exception if an ImportError arises at this point.

Also, we should state somewhere that `dask` is "an optional requirement of Devito, to..." ( @mlange05 ?)